spec/features/user_visits_genre_index_clicks_genre_name_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+describe 'User visits genres index' do
+  context 'clicks on genre name' do
+    it 'should show related movies' do
+      director = Director.create!(name: 'Someone')
+      movie_1 = Movie.create!(director_id: director.id, title: "Guardians of the Galaxy", description: "Fun space movie", rating: 4)
+      genre1 = movie_1.genres.create(name: 'Action')
+      movie_2 = genre1.movies.create(director_id: director.id, title: "Suicide Squad", description: "scary space movie", rating: 2)
+
+      visit genres_path
+
+      click_on genre1.name
+      expect(page).to have_content(movie_1.title)
+      expect(page).to have_content(movie_2.title)
+    end
+
+    it 'should show average rating for movies in this genre' do
+      director = Director.create!(name: 'Someone')
+      movie_1 = Movie.create!(director_id: director.id, title: "Guardians of the Galaxy", description: "Fun space movie", rating: 4)
+      genre1 = movie_1.genres.create(name: 'Action')
+      movie_2 = genre1.movies.create(director_id: director.id, title: "Suicide Squad", description: "scary space movie", rating: 1)
+      movie_3 = genre1.movies.create(director_id: director.id, title: "Another Sci-Fi Movie", description: "There's some science fiction involved", rating: 4)
+      movie_4 = genre1.movies.create(director_id: director.id, title: "Yet Another Sci-Fi Movie", description: "There's some science fiction involved", rating: 4)
+
```

Comment:
would have been good to create a movie or two with a different genre to make sure it wasn't being calculated as part of the average, too